target/linux/layerscape/base-files/lib/upgrade/platform.sh

```diff
@@ -107,15 +107,29 @@ platform_do_upgrade_traverse_nandubi() {
 	nand_do_upgrade "$1" || (echo "Upgrade failed, setting bootsys ${bootsys}" && fw_setenv bootsys $bootsys)
 
 }
-platform_copy_config() {
-	local partdev parttype=ext4
+platform_copy_config_sdboot() {
+	local diskdev partdev parttype=ext4
+
+	export_bootdevice && export_partdevice diskdev 0 || {
+		echo "Unable to determine upgrade device"
+		return 1
+	}
 
 	if export_partdevice partdev 1; then
-		mount -t $parttype -o rw,noatime "/dev/$partdev" /mnt
+		mount -t $parttype -o rw,noatime "/dev/$partdev" /mnt 2>&1
 		cp -af "$UPGRADE_BACKUP" "/mnt/$BACKUP_FILE"
 		umount /mnt
 	fi
 }
+platform_copy_config() {
+	local board=$(board_name)
+
+	case "$board" in
+	*sdboot*)
```

Comment:
I'd prefer to list the devices explicitly, as using wildcards makes it impossible to grep for board names.